bindings/nodejs/lib/wallet/account.ts

```diff
@@ -121,12 +121,12 @@ export class Account {
      * @param burn The outputs to burn
      * @param transactionOptions The options to define a `RemainderValueStrategy`
      * or custom inputs.
-     * @returns The resulting transaction.
+     * @returns The prepared transaction.
      */
     async prepareBurn(
         burn: Burn,
         transactionOptions?: TransactionOptions,
-    ): Promise<Transaction> {
+    ): Promise<PreparedTransaction> {
```

Comment:
Not sure we can do that, breaking change

Reply:
How can it break anything, if nothing can work with it, like it is without this change?

Reply:
What would happen if I call this function now? Does it crash ?